addons/sys_core/fnc_shouldBeMuted.sqf

```diff
@@ -0,0 +1,28 @@
+#include "script_component.hpp"
+/*
+ * Author: ACRE2Team
+ * Check if a remote TS Client Should be muted
+ *
+ * Arguments:
+ * 0: Unit
+ * 1: Remote TS Id
+ * 2: Dynamic Position
+ *
+ * Return Value:
+ * Unit Should Be Muted
+ *
+ * Example:
+ * [] call acre_sys_core_fnc_shouldBeMuted
+ *
+ * Public: No
+ */
+params ["_remoteUser", "_remoteTs3Id", "_dynamicPos"];
+
+if (_remoteUser in EGVAR(sys_godmode,speakingGods)) exitWith { false };
+if (ACRE_IS_SPECTATOR && {_remoteTs3Id in ACRE_SPECTATORS_LIST}) exitWith { false };
```

Comment:
could replace 22/23, but this is subjective if better
```suggestion
if (_remoteTs3Id in ACRE_SPECTATORS_LIST) exitWith { !ACRE_IS_SPECTATOR };
```

Reply:
i like this gonna use this. did thought of how to improve this but did not had this idea.